reference/clinic.v1.yaml

```diff
@@ -1051,6 +1051,45 @@ paths:
       tags:
         - Clinics
         - Internal
+  '/v1/clinics/{clinicId}/membership_restrictions':
+    parameters:
+      - $ref: '#/components/parameters/clinicId'
+    get:
+      summary: List Membership Restrictions
+      tags:
+        - Clinics
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/MembershipRestrictions'
+      operationId: ListMemebershipRestrictions
+      description: |-
+        Returns a list of the membership restrictions that will be evaluated when a user joins a clinic.
+
+        Only clinic admins can access this endpoint.
+    put:
+      summary: Update Membership Restrictions
+      operationId: UpdateMembershipRestrictions
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/MembershipRestrictions'
+      description: An internal endpoint to update clinic membership restrictions
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/MembershipRestrictions'
+      x-internal: true
```

Comment:
FYI, minor note: there's no need for `x-internal: true`. I had used it for just tagging things for internal use - the `tags: Internal` is what is actually used by the publish script to segment the APIs when publishing to Stoplight.